src/services/class.service.ts

```diff
@@ -323,6 +323,20 @@ export class ClassService {
     return `You have successfully been removed from the ${categoryType} category.`;
   }
 
+  private _tokenizeClassChannelName(channelName: string) {
+    const tokenizingRegex = /^([a-z]{3})([0-9]{4}h?)_([a-z]+)$/;
```

Comment:
This [already exists](https://github.com/cs-discord-at-ucf/lion/blob/5b13a8c7ba254e646bf8659c0a011820153ead34/src/app/commands/delclasschans.command.ts#L7) somewhere. Could you remove the one in delclasschans, and put yours in `constants.ts` since yours is slightly better